sample/src/main/scala/memory.scala

```diff
@@ -0,0 +1,52 @@
+package scala.scalajs.wasm
```

Comment:
The tests are failing because these code are not available outside the `sample` project.
However, I'm not sure where to put those library code in our project 🤔